### PR TITLE
Add Tag list basic functionality

### PR DESCRIPTION
**Pull request by oleksii-demedetskyi**
Implemented with using flow layout
:construction: 
Final result is not satisfying. Tags need to be relayouted.

### CODE REVIEW

**Comment by oleksii-demedetskyi** [comment]
Current state of UI.

**Comment by oleksii-demedetskyi** [comment]
Should we do generating all ids from storyboard automatically?
https://github.com/kluivers/storyboard-constants

File: Robe Guard/sources/tag selection/tag list view/RBGTagListCollectionViewController.m [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+@property (nonatomic, strong) RBGTagListCollectionViewCellRuler* cellRuler;
+
+@end
+
+@implementation RBGTagListCollectionViewController
+
+- (UINib*)nibForTagCell
+{
+    NSDictionary* nibMap = [self.collectionView valueForKey:@"_cellNibDict"];
```
Comment (xNekOIx) [inline Robe Guard/sources/tag selection/tag list view/RBGTagListCollectionViewController.m:37]: Is it some Blood Magic? :smiling_imp:

**Comment by xNekOIx** [comment]
:accept:

**Pull request merged**